app/components/blacklight/search_context/server_item_pagination_component.rb

```diff
@@ -20,7 +20,7 @@ def render?
       end
 
       def display_separator
-        '|' unless total == 1
+        t('views.pagination.separator') unless total == 1
```

Comment:
Do we really want to configure the delimiter separately for each language that we want to support?  This seems to me like something better suited for Blacklight::Config rather than an i18n.